Watchman.Discord/Areas/Users/Controllers/WarnsController.cs

```diff
@@ -43,10 +44,19 @@ public async Task AddWarn(AddWarnCommand command, Contexts contexts)
             await messageService.SendResponse(x => x.UserHasBeenWarned(contexts.User.Name, mentionedUser.Name, command.Reason));
         }
 
-        public async Task Warns(WarnsCommand command, Contexts contexts)
+        public Task Warns(WarnsCommand command, Contexts contexts)
         {
-            var mentionedUser = command.User == 0 ? contexts.User : await this._usersService.GetUserByIdAsync(contexts.Server, command.User);
-            await this._warnService.GetWarns(command, contexts, mentionedUser, contexts.Server.Id);
+            var messageService = _messagesServiceFactory.Create(contexts);
+            var mentionedUser = command.User == 0 ? 
+                    contexts.User 
+                        : 
+                    this._usersService.GetUserByIdAsync(contexts.Server, command.User).GetAwaiter().GetResult();
```

Comment:
coś tu się stało z formatowaniem

Reply:
to specjalnie, bo długie było strasznie, ale skoro się nie przyjęło to ok xD